main/index.js

```diff
@@ -146,10 +146,6 @@ app.on('ready', async () => {
       /* eslint-enable no-console */
   }
 
-  if (config._is_beta === true) {
-    // XXX do something like delete all the measurements on a fresh launch.
```

Comment:
We may want to keep this and add some extra logic to do some cleanup if we notice the `_is_beta` key inside of the config file.

This is mostly for some possible inconsistencies between the `probe-cli` version of the beta and what we are going to be shipping in the stable release.

@bassosimone may have some thoughts about this.

If this is not necessary we can just drop it directly, but let's also remember to drop that key from the config file.

Reply:
I have restored this piece for now. To be able to land this in the release branch.